code/modules/grey_market/illegalradio.dm

```diff
@@ -0,0 +1,379 @@
+#define MAIN 1
+#define SELLING 2
+#define DELIVERY 3
+
+var/list/global_illegal_radios = list()
+
+/proc/buzz_grey_market() //Updates and buzzes all illegalradios so the screen is current.
+	for(var/obj/item/device/illegalradio/radio in global_illegal_radios)
+		if(radio.notifications)
+			var/mob/living/holder = get_holder_of_type(radio, /mob/living)
+			if(holder)
+				radio.interact(holder)
+				to_chat(holder,"<span class='info'>You feel \the [radio] buzz.</span>")
+
+/obj/item/device/illegalradio/nanotrasen/New()
+	qdel(src) //Removed, but mapping issues so xd
```

Comment:
Didn't you purge it from all the maps?

Reply:
Last I heard, the beacon is still in Box's bridge.

Reply:
We seem to be generating some confusion here with word tenses.

* It is true that the radio was purged from all maps.
* The radio is still on all maps on live, because the purge is in this PR.

Reply:
why would you leave the item coded in to selfdestruct if it doesn't exist anymore?

Reply:
Don't do this. Just delete the item from the code.

Reply:
That's going to result in compilation errors for the maps. I'll remove it soon (tm), as map gonflicts are causing a buncha shit and I'll have Kharkov do it.

Reply:
Honestly you could just do the mapping portion as its own PR and remove the special uplink there. Not for atomicity but to make it easier to get past mapping conflicts. This kind of thing is common in major PRs, like when deity did Viro and had to do mapping for the new machines.